### ISSUE TITLE
[3.8.1] `ionic cordova build android ... --alias=foo` spams lines instead of running spinner in place

### ISSUE THREAD

**Issue by janpio** [ionic-team/ionic-cli#2649]
Running `ionic cordova build android --no-build --prod --release -- -- --keystore=lala.keystore --alias=foo` (the `--alias=...` seems to be the trigger) on a Mac just gave me this output:

```
Running app-scripts build: --prod

[17:36:11]  build prod started ... 
[17:36:11]  clean started ... 
[17:36:11]  clean finished in 32 ms 
[17:36:11]  copy started ... 
[17:36:11]  ngc started ... 
[17:36:24]  ngc finished in 13.19 s 
[17:36:24]  preprocess started ... 
[17:36:24]  deeplinks started ... 
[17:36:25]  deeplinks finished in 274 ms 
[17:36:25]  optimization started ... 
[17:36:25]  copy finished in 14.31 s 
[17:36:39]  optimization finished in 13.89 s 
[17:36:39]  preprocess finished in 14.16 s 
[17:36:39]  webpack started ... 
[17:36:46]  webpack finished in 7.84 s 
[17:36:46]  sass started ... 
[17:36:46]  uglifyjs started ... 
[17:36:48]  sass finished in 1.49 s 
[17:36:48]  cleancss started ... 
[17:36:50]  cleancss finished in 1.67 s 
[17:37:13]  uglifyjs finished in 27.02 s 
[17:37:13]  postprocess started ... 
[17:37:14]  postprocess finished in 74 ms 
[17:37:14]  lint started ... 
[17:37:14]  build prod finished in 62.47 s 
> ionic cordova prepare
✔ cordova prepare - done!

⠙ cordova build android 
⠹ cordova build android
⠸ cordova build android
⠼ cordova build android
⠴ cordova build android
⠦ cordova build android
⠧ cordova build android
⠇ cordova build android
⠏ cordova build android
⠋ cordova build android 
⠙ cordova build android
[...]
```

ionic info:
```
cli packages: (/Users/sujan/stromzaehler/node_modules)

    @ionic/cli-utils  : 1.8.1
    ionic (Ionic CLI) : 3.8.1

global packages:

    Cordova CLI : 7.0.1 

local packages:

    @ionic/app-scripts : 2.0.2
    Cordova Platforms  : android 6.2.3 ios 4.4.0
    Ionic Framework    : ionic-angular 3.5.3

System:

    Android SDK Tools : 26.0.2
    ios-deploy        : 1.9.1 
    ios-sim           : 5.0.1 
    Node              : v7.8.0
    npm               : 4.2.0 
    OS                : OS X El Capitan
    Xcode             : Xcode 8.2.1 Build version 8C1002 

```

**Comment by dwieeb** [comment]
It will do this if for some reason it needs to word wrap the command once it exceeds the terminal width, but `cordova build android` is not a long command.

**Comment by janpio** [comment]
I actually first posted a censored version without the params.

Then I investigated and found out which parameter triggered this, then edited my post to include them in the command on the top but forget to add it back at the bottom.

So yes, there would be more here than `cordova build android` - sorry. More like `cordova build android --no-build --prod --release -- -- --keystore=lala.keystore --alias=foo`
